test/system/410-selinux.bats

```diff
@@ -74,6 +74,15 @@ function check_label() {
     check_label "--security-opt label=level:s0:c1,c2" "container_t" "s0:c1,c2"
 }
 
+@test "podman selinux: inspect kvm labels" {
+    skip_if_no_selinux
+    skip_if_remote "runtime flag is not passed over remote"
+
+    run_podman create --runtime=kata --name myc $IMAGE
```

Comment:
Error: default OCI runtime "kata" not found: invalid argument

Reply:
If you touch /usr/bin/kata-runtime does the test work?

Reply:
Almost. This works: `ln -s true /usr/bin/kata-runtime`. Point is, this new test is going to fail in CI (and Fedora/RHEL gating) because of the missing runtime. It might be prudent to:
```bash
    if ! -x /usr/bin/kata-runtime; then
        skip "kata runtime unavailable"
    fi
```

(And I forgot to mention: your rootless changes work fine. Is that a new feature, or was I mistaken when I disabled those for rootless?)

Reply:
I think it is a fairly new feature.

BTW, I think I can fake this test out by doing

ln -s /bin/true  $TMPDIR/kata-runtime

Then set the runtime to this, which should trigger the code to do the right thing, once my other kata PR gets merged.